src/components/SchemaRow/SchemaRow.tsx

```diff
@@ -55,6 +55,7 @@ export const SchemaRow: React.FunctionComponent<SchemaRowProps> = ({ schemaNode,
   const isBrokenRef = typeof refNode?.error === 'string';
 
   const childNodes = React.useMemo(() => calculateChildrenToShow(typeToShow), [typeToShow]);
+  const combiner = anyOfOrOneOf(schemaNode);
```

Comment:
A node should have a combiner property you could use
https://github.com/stoplightio/json-schema-tree/blob/master/src/nodes/RegularNode.ts#L40

```suggestion
  const combiners = isRegularNode(schemaNode) && schemaNode.combiners?.length ? schemaNode.combiners[0]: null;
```
I haven't tested it, but I believe it should work fine.

Reply:
Ah, my editor wasn't giving me that in auto-complete, I assume because I wasn't using `isRegularNode`. TIL!
<img width="189" alt="Screen Shot 2021-11-04 at 11 41 21 AM" src="https://user-images.githubusercontent.com/3588798/140381781-d85aa8cd-1c88-4f00-8870-04dcdb76a476.png">